session/handler.go

```diff
@@ -124,6 +126,13 @@ type toSession struct {
 	//
 	// in: header
 	Cookie string `json:"Cookie"`
+
+	// Return the session additional in token format like a JWT
+	//
+	// The value of this parameter has to be a valid, configured Ory Session token template.
+	//
+	// in: query
+	Tokenize string `json:"tokenize"`
```

Comment:
```suggestion
	TokenizeWith string `json:"tokenize_with"`
```

Just `tokenize` sounds like it's a boolean value.

Reply:
That's true! I'm also not super happy with the name right now!

- tokenize_with
- tokenize_to
- tokenize_template

Which one of these should we use?

Reply:
I opted for `tokenize_to`